src/projects/console-app/environments/environment.ts

```diff
@@ -0,0 +1,6 @@
+export const environment = {
+    apiHost: 'http://localhost:3000',
+    production: false,
+    preAuth: false,
+    preAuthPath: '/preauth'
+};
```

Comment:
This file was generated by `set-env.ts`

Reply:
yes, we're using generated environment variables. its easier to manage and use to deploy multiple branches. we can update the file entry in .gitignore, we don't need to commit this.